src/callbacks_step/glm_speed.jl

```diff
@@ -0,0 +1,72 @@
+
+"""
+    GlmSpeedCallback(; glm_scale=0.5, cfl_scale=1.0)
+
+Update the divergence cleaning wave speed c_h according to the time step
+computed in StepsizeCallback for the ideal GLM-MHD equations.
+"""
+mutable struct GlmSpeedCallback{RealT<:Real}
+  glm_scale::RealT
+  cfl_scale::RealT
+end
```

Comment:
```suggestion
struct GlmSpeedCallback{RealT<:Real}
  glm_scale::RealT
  cfl_scale::RealT
end
```
You do never modify a `GlmSpeedCallback` after creating it, do you? In that case, it's better (for the compiler) to make it a non-mutable `struct`.